src/data/wallets/wallets.yml

```diff
@@ -58,10 +62,10 @@
     - core_software
   highlight: true
   links:
-    - title: GitHub
+    - title: Downloads and release notes
+      url: https://github.com/decred/decred-binaries/releases/tag/v1.8.1
+    - title: Installer (advanced)
```

Comment:
I'm not sure about this. We have previously considered the loose binaries to be the advanced version because they need more work to configure. dcrinstall was the laymans version because it handles hash verification and config setup automatically.

Reply:
Sorry I forgot to add context. I got the idea of de-prioritizing dcrinstall from this [chat](https://matrix.to/#/!mlRZqBtfWHrcmgdTWB:decred.org/$iZHzwEVq_B4cqPjIITAXYAZiLc1ax_itr1Q8i-G2jTs) in #dex:

> dcrinstall needs a dedicated maintainer or else needs to be deprecated.

> tbh i think dcrinstall has out-lived its usefulness. It was the correct tool for the job at the time, but now that we have decrediton and dex its kind of pointless

> it was originally built to make CLI tools accessible for non-technical people back when we didnt have a decent GUI wallet

> like 90% of the support burden was just helping people to place and modify dcrd/dcrwallet config files, and dcrinstall fixed that

> but in 2023 the kind of people who need that kind of support should prob just use decrediton instead

It happened after someone installed an outdated version of dexc via dcrinstall. I figured those are good points that we can reflect on Wallets.

Also its [release page](https://github.com/decred/decred-release/releases/tag/v1.8.1) has no release notes or instructions what to do with the files, not even a link to decred-binaries. So we'd be sending users to no-information page as a recommended option. OTOH [decred-binaries](https://github.com/decred/decred-binaries/releases/tag/v1.8.1) links to dcrinstall for installing the CLI tools. Hash and sig verification needs to be done for dcrinstall files too (chicken and egg problem).

That said I can swap the recommended/advanced links back to not block on this and discuss it separately.

Reply:
It isn't included in the release notes, but documentation for dcrinstall does exist [here](https://docs.decred.org/wallets/cli/cli-installation/) and [here](https://github.com/decred/decred-release) .

I think this needs a wider discussion, could you please pull this change out of the PR so we can merge the rest of the improvements?

Reply:
Yeah let's get more feedback on it. I'll do that in a separate PR.

Changed:

- Returned the order to put dcrinstall first.
- Added [CLI guide](https://docs.decred.org/wallets/cli/cli-installation/) link. It is a good entry point that explains what will be installed and where.
- Changed last link to "Release notes + standalone" to hint that it's also a download option (I always use decred-binaries personally).
- Updated `en.json` text.

Had to squash to update the commit and PR messages too. Changes since last iteration [here](https://github.com/decred/dcrweb/compare/2980098e69fc089f854e3b29f5369589d3ba0887..df018beae1b119d4ad15cd05b3d920bcb6be923d).

I wonder if the CLI panel is overloaded now?